src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VersionFileCodeFlowUpdater.cs

```diff
@@ -483,7 +483,7 @@ private async Task<List<DependencyUpdate>> BackflowDependenciesAndToolset(
                 continue;
             }
 
-            _logger.LogInformation("Asset {assetName} is not part of the build, not updated in the repo and not updated in the VMR. Skipping", assetName);
+            _logger.LogDebug("Asset {assetName} is not part of the build, not updated in the repo and not updated in the VMR. Skipping", assetName);
```

Comment:
Wouldn't this be useful for debugging missing assets in a running service? i.e why downgrade it in priority to debug?

Reply:
All of the other ones are more important, this one was just a lot of noise in the logs actually.

Overall it was a bit of a problem for the `darc vmr backflow` that the logs were quite nicely concise and then this threw in a full screen of uninteresting stuff.

Reply:
@oleksandr-didyk just for the record - for a real VMR build, this was 3000k lines